### DISCUSSION TITLE
Play FEL on phone with 12-bit panel?

### DISCUSSION THREAD

**Discussion by Amsitram** [quietvoid/dovi_tool#231] [Q&A] [answered]
Is it possible to display a 12-bit stream from a FEL on the 12-bit screen of my phone (Xiaomi 12X)?

**Comment by quietvoid** [comment, marked as answer]
Probably not.

**Comment by NeedsMoar** [comment]
Playing it directly no, like quietvoid said.

If you map the whole mess to a 12bpc 4:2:2 and re-encode, and the phone can handle playing 12 bit HEVC then maybe.
I don't know what benefit you'd actually get.  According to the DXOMark tests it somehow fails to cover all of DCI-P3 in HDR mode when any 10-bit panel and most 8-bit + FRC panels can manage that...   a 12 bit panel should easily be able to cover BT.2020.  I haven't found a proper colorimeter test showing it can actually display 68b colors yet.  DXOMark rates it in the ~130s but that's not high-end for them.  I'm kinda surprised they failed to test that part.  

I'm really not sure what the difficulty is with LED-based technologies since the brightness control is analogue, maybe they're just using up a huge stock of 10-bit DACs, I dunno.  I'd expect we have accurate enough voltage control by now.  It's not IPS where switching between 1024 polarization angles was already a nightmare and going higher was more or less impossible.

I recall Eizo or somebody advertising their 10-bit panels based on the depth of the internal LUTs and calling it a 68B color display.  It was a dumb stunt but I'm sure it got some more photographers on board with their $4000 panels.

**Reply by Amsitram** [reply]
I don't think it's worth re-encoding, in any case.
I haven't found any test that confirms the 68 billion colors either. DisplayMate doesn't mention it.
Most Macs between 2018 and 2021 also use a 10-bit LUT when they only display 8-bit, but they don't call themselves 10-bit for that (just like the fact that they support HDR and Dolby Vision, without putting it in the specs).
